core/src/main/resources/filodb-defaults.conf

```diff
@@ -184,6 +184,12 @@ filodb {
     # Group by Cardinality limit
     group-by-cardinality-limit = 1000
 
+    # Maximum result size returnable from any ExecPlan::execute call (including nested calls)
+    result-byte-limit = 18MB
```

Comment:
You may want to document that this setting needs to be tweaked in tandem with Akka message size limit. Increasing just this will not do it. Speaking of which, the Akka message size limit of 25MB can also be reduced to 18MB now.

Reply:
done!